src/virtual-select.js

```diff
@@ -2332,7 +2332,9 @@ export class VirtualSelect {
 
     if (!isSilent) {
       DomUtils.dispatchEvent(this.$ele, 'afterClose');
-      this.focus();
+      if ((this.initialSelectedValue && this.initialSelectedValue.length === 0) || this.selectedValues.length > 0) {
```

Comment:
@bmarcelino-fe could you please add a comment to explain the goal of this validation? Thank you

Reply:
@bmarcelino-fe could you please add a comment to explain the goal of this validation? Thank you